### DISCUSSION TITLE
Application not verified for Github Student Development Pack

### DISCUSSION THREAD

**Discussion by sukhpreet37140** [community/community#106862] [GitHub Education]
I have applied for GitHub Student Development Pack on Feb 06 but got this message.

**You were last verified as a null on Feb 6, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.**

Now, I have added my repositories and personal information(profile picture email, description etc). How much time will it take to reverify? **Is there any way to request for reverification?**

**Comment by pH-7** [comment]
I wouldn’t be too worried to the message you received. It seems to happen to “most” (if not all) people signing up to the program. Someone else already asked about it here https://github.com/orgs/community/discussions/103082

I don’t think you can request a re-verification. It would be best to wait a few more days. The waiting time is usually around 5 days.
